packages/fxa-shared/test/subscriptions/metadata.ts

```diff
@@ -0,0 +1,178 @@
+import { expect } from 'chai';
+
+import { Plan, ProductMetadata } from '../../subscriptions/types';
+import {
+  metadataFromPlan,
+  productDetailsFromPlan,
+} from '../../subscriptions/metadata';
+
+const NULL_METADATA = {
+  productSet: null,
+  productOrder: null,
+  emailIconURL: null,
+  webIconURL: null,
+  upgradeCTA: null,
+  downloadURL: null,
+};
+
+const PLAN: Plan = {
+  plan_id: 'plan_8675309',
+  product_id: 'prod_8675309',
+  product_name: 'Example product',
+  currency: 'usd',
+  amount: 599,
+  interval: 'month' as const,
+  interval_count: 1,
+};
+
+describe('subscriptions/metadata', () => {
```

Comment:
This is a conversion of the `store/utils.test.tsx` Jest test suite from fxa-payments-server to mocha/chai in fxa-shared